docs/user/library/render/gtrenderer.rst

```diff
@@ -38,7 +38,7 @@ consider the following ideas:
 * Experiment with different Java 2D graphics settings such as
   anti-aliasing
 * Use background threads to draw tiles, and allowing your swing control
-  to pull the tiles onto the screen for a smooth "slippy" map
+  to pull the tiles onto the screen for a smooth "slippery" map
```

Comment:
I think this change is incorrect. Its called a "slippy map".

Reply:
hah!  yes, most definitely slippy map.  Thanks for the catch.  I will follow up with a pull request to fix that one.